packages/amplify-util-uibuilder/src/commands/generateComponents.ts

```diff
@@ -42,7 +43,7 @@ export const run = async (context: $TSContext): Promise<void> => {
 
     const generatedResults = {
       component: generateUiBuilderComponents(context, componentSchemas.entities, dataSchema),
-      theme: generateUiBuilderThemes(context, themeSchemas.entities),
+      theme: themeSchemas.entities.length > 0 ? generateUiBuilderThemes(context, themeSchemas.entities) : [generateDefaultTheme(context)],
```

Comment:
`commands/generateComponents.ts` doesn't seem like the best place for this logic to live.

I would rather see `generateUiBuilderThemes` fall back to the default theme if no `themeSchemas.entities` are present. That's one less exported method to reason about.

Reply:
@jhockett I made the update